udata/core/pages/api.py

```diff
@@ -0,0 +1,46 @@
+from flask import request
+from flask_login import current_user
+
+from udata.api import API, api
+from udata.api_fields import patch
+
+from .models import Page
+
+ns = api.namespace("pages", "Pages related operations (beta)")
+
+common_doc = {"params": {"page": "The page ID or slug"}}
+
+
+@ns.route("/", endpoint="pages")
+class PagesAPI(API):
+    @api.secure
+    @api.doc("create_page", responses={400: "Validation error"})
+    @api.expect(Page.__write_fields__)
+    @api.marshal_with(Page.__read_fields__, code=201)
+    def post(self):
```

Comment:
I guess it may come from cdata logic, but playing on https://demo.data.gouv.fr/admin/site/edito/datasets, reuses & API, I ended up with 18 object counts in the end.
It seemed that multiple times, it created new objects instead of saving it to the existing one.